python/minmaxmap.py

```diff
@@ -14,9 +16,24 @@
     line = line.strip()
     line = line.split()
     slice_file = os.path.join(line[0])
+    slice = None
 
     #load nifti image
-    slice = nib.load(slice_file)
+    if not os.path.exists(slice_file):
```

Comment:
I had originally implemented it as such, but it tries to read /user/username/hdfs://host:port/user/username/filename instead of just hdfs://host:port/user/username/filename. Have not yet found a way around it.